modules/rostests/apitests/com/com_apitest.c

```diff
@@ -30,488 +31,505 @@ typedef struct _KNOWN_INTERFACE
+    { ID_NAME(IID_IFolderProperties),              RegisteredNever  },
+    { ID_NAME(IID_IFolderWithSearchRoot),          RegisteredNever  },
+    { ID_NAME(IID_ILocalizableItemParent),         RegisteredNever  },
+    { ID_NAME(IID_IPrinterFolder),                 RegisteredNever  },
+    { ID_NAME(IID_IPropertyControl),               RegisteredNever  },
+    { ID_NAME(IID_IPropertyControlBase),           RegisteredNever  },
+    { ID_NAME(IID_IPropertyControlSite),           RegisteredNever  },
+    { ID_NAME(IID_IRegItemCustomAttributes),       RegisteredNever  },
+    { ID_NAME(IID_IRegItemCustomEnumerator),       RegisteredNever  },
+    { ID_NAME(IID_IScope),                         RegisteredNever  },
+    { ID_NAME(IID_IScopeItem),                     RegisteredNever  },
+    { ID_NAME(IID_ITaskCondition),                 RegisteredNever  },
+    { ID_NAME(IID_ITaskConditionCombiner),         RegisteredNever  },
+    { ID_NAME(IID_ITaskConditionInit),             RegisteredNever  },
+    { ID_NAME(IID_ITransferProvider),              RegisteredNever  },
+    { ID_NAME(IID_IAssociationArrayOld),           RegisteredNever  },
+    { ID_NAME(IID_IPinnedListOld),                 RegisteredNever  },
+    { ID_NAME(IID_IPinnedList),                    RegisteredNever  },
+    { ID_NAME(IID_IAttachmentExecute),             RegisteredNever  },
+    { ID_NAME(IID_IComponentData),                 RegisteredNever  },
+    { ID_NAME(IID_IConsole),                       RegisteredNever  },
+    { ID_NAME(IID_IConsole2),                      RegisteredNever  },
+    { ID_NAME(IID_IConsoleNameSpace),              RegisteredNever  },
+    { ID_NAME(IID_IConsoleNameSpace2),             RegisteredNever  },
+    { ID_NAME(IID_IPropertySheetCallback),         RegisteredNever  },
+    { ID_NAME(IID_IPropertySheetProvider),         RegisteredNever  },
+    { ID_NAME(IID_IExtendPropertySheet),           RegisteredNever  },
+    { ID_NAME(IID_IExtendPropertySheet2),          RegisteredNever  },
+    { ID_NAME(IID_IHeaderCtrl),                    RegisteredNever  },
+    { ID_NAME(IID_IToolbar),                       RegisteredNever  },
+    { ID_NAME(IID_IImageList_mmc),                 RegisteredNever  },
+    { ID_NAME(IID_IConsoleVerb),                   RegisteredNever  },
+    { ID_NAME(IID_ISnapInAbout),                   RegisteredNever  },
+    { ID_NAME(IID_ICertificateManager),            RegisteredNever  },
+    { ID_NAME(IID_IEnumNetCfgBindingInterface),    RegisteredNever  },
+    { ID_NAME(IID_IEnumNetCfgBindingPath),         RegisteredNever  },
+    { ID_NAME(IID_INetCfgBindingInterface),        RegisteredNever  },
+    { ID_NAME(IID_INetCfgBindingPath),             RegisteredNever  },
 };
 static const INT KnownInterfaceCount = RTL_NUMBER_OF(KnownInterfaces);
 
+#define ValidClassForVersion(pClass, version) \
+    (((pClass->MinClassNTDDIVersion) <= version) && ((pClass->MaxClassNTDDIVersion) >= version))
```

Comment:
```suggestion
    ((pClass)->MinClassNTDDIVersion <= (version) && (pClass)->MaxClassNTDDIVersion >= (version))
```
parenthesizing the macro parameters (and same for the other instance below).